### DISCUSSION TITLE
Add network preference like Wifi

### DISCUSSION THREAD

**Discussion by chpego** [seaky/nspanel_pro_tools_apk#141] [Ideas]
What do you think ?

**Comment by chpego** [comment]
I'd like to be able to send a command via adb to add a Wifi network - too long to type by hand on the small screen.

**Comment by seaky** [comment]
Sorry I do not really understand your request, 
To use adb you definitelly need a preconfigured wifi connection. ADB via usb is only needed for initial rooting.
